.github/CONTRIBUTING.md

```diff
@@ -2,7 +2,12 @@
 
 If you discover issues, have ideas for improvements or new features,
 please report them to the [issue tracker][1] of the repository or
-submit a pull request. Please, try to follow these guidelines when you do so.
+submit a pull request.
+
+We consider reporting issues and submitting ideas full contribution to

```

Comment:
so should we invoke our native speaker correspondent? no idea who that would be tho ;)